test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -93,41 +112,92 @@ func (a *acmeIssuerProvisioner) delete(f *framework.Framework, ref cmmeta.Object
 // - pebble
 // - a properly configured Issuer resource
 
-func (a *acmeIssuerProvisioner) createHTTP01(f *framework.Framework) cmmeta.ObjectReference {
+func (a *acmeIssuerProvisioner) createHTTP01Issuer(f *framework.Framework) cmmeta.ObjectReference {
 	a.deployTiller(f, "http01")
 
-	By("Creating an ACME HTTP01 issuer")
+	By("Creating an ACME HTTP01 Issuer")
 	issuer := &cmapi.Issuer{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "acme-issuer-http01",
+			GenerateName: "acme-issuer-http01-",
 		},
-		Spec: cmapi.IssuerSpec{
-			IssuerConfig: cmapi.IssuerConfig{
-				ACME: &cmacme.ACMEIssuer{
-					Server:        addon.Pebble.Details().Host,
-					SkipTLSVerify: true,
-					PrivateKey: cmmeta.SecretKeySelector{
-						LocalObjectReference: cmmeta.LocalObjectReference{
-							Name: "acme-private-key-http01",
-						},
+		Spec: a.createHTTP01IssuerSpec(),
+	}
+
+	issuer, err := f.CertManagerClientSet.CertmanagerV1alpha2().Issuers(f.Namespace.Name).Create(issuer)
+	Expect(err).NotTo(HaveOccurred(), "failed to create acme HTTP01 issuer")
+
+	return cmmeta.ObjectReference{
+		Group: cmapi.SchemeGroupVersion.Group,
+		Kind:  cmapi.IssuerKind,
+		Name:  issuer.Name,
+	}
+}
+
+func (a *acmeIssuerProvisioner) createHTTP01ClusterIssuer(f *framework.Framework) cmmeta.ObjectReference {
+	a.deployTiller(f, "http01")
+
+	By("Creating an ACME HTTP01 ClusterIssuer")
+	issuer := &cmapi.ClusterIssuer{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "acme-cluster-issuer-http01-",
```

Comment:
Again, how do we clean this up if the name is randomly generated? 😄

Reply:
Ah, I guess via the ObjectReference returned in this case.. but the same applies for the others where you've created Secrets at least 😄